src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -177,6 +177,11 @@
     <PropertyGroup>
         <PackageBuildNumberOverride Condition="'$(PackageBuildNumberOverride)' == '' and '$(_DerivedPackageNumber)' != ''">$(_DerivedPackageNumber)</PackageBuildNumberOverride>
     </PropertyGroup>
+    <!-- we don't yet have runtime packages for these RIDs so exclude them for now -->
+    <ItemGroup>
+      <ExcludedRuntimes Include="fedora.23-x64" />
```

Comment:
@weshaggard Gotcha, makes sense.  I just opened a PR to support building CoreFX on the other distros, so maybe we can remove all this stuff later tomorrow or early next week.  Makes sense to do this now to unblock you.